game_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rspec'
+require_relative './game'
+require_relative './author'
+
+RSpec.describe 'Game' do
+  describe '#can_be_archived?' do
+    it 'returns true if parents_results && last_played_duration else false' do
+      multiplayer = true
+      last_played_at = '2021-06-01'
+      publish_date = '2020-01-01' # Replace with a valid publish_date
+
+      results = Game.new(nil, nil, nil, nil, nil, publish_date, nil, multiplayer, last_played_at)
+      final = results.can_be_archived?
+      expect(final).to eql(false)
+    end
+  end
+end
```

Comment:
- [ ] This test is failing as can be seen below 👇 Please make it green ✔️ 
![image](https://github.com/MuhammadUsmanAslam/catalog_of_my_things/assets/46839548/384f0bac-972f-4a43-a368-c8c4f7de43a7)
#### NOTE: running `rspec` doesn't execute all of the tests because they are not all in the `./spec` folder; Only the 6 tests in the `./spec` folder runs. I had to specifically run `rspec game_spec.rb` for me to see that this test is failing.

- [ ] Please place all of your tests in the `./spec` folder to improve structure, collaboration and also ease of running tests 😎